### ISSUE TITLE
Feature Request: Implement Continuous Bernoulli and Continuous Categorical distributions

### ISSUE THREAD

**Issue by wdkrnls** [twolodzko/extraDistr#21]
I'm a big fan of your package 👍. I was curious if you might consider adding two new exponential family distributions for percent or compositional data: the Continuous Bernoulli and Continuous Categorical which have been recently added into [tensor flow](https://www.tensorflow.org/probability/api_docs/python/tfp/edward2/ContinuousBernoulli). The paper for Continuous Categorical is [here](https://arxiv.org/abs/2002.08563).

**Comment by twolodzko** [comment]
Thanks, it's nice to hear that you found it helpful. Unfortunately, currently I'm only maintaining the package, so it is unlikely that I'll add any new features. However I'll be happy to receive PR, if you, or someone else, would be willing to contribute.

**Comment by wdkrnls** [comment]
Fair enough. Thanks for maintaining :+1:. The Continuous Bernoulli is pretty simple to code up naively in base R, but I am not too familiar with Rcpp and am rusty with numerical tricks to minimize floating point issues. The Continuous Categorical distribution is more challenging, but sampling on the Simplex might benefit a lot from Rcpp AFAICT.
